mcs/class/referencesource/mscorlib/system/threading/threadpool.cs

```diff
@@ -1145,10 +1149,11 @@ private bool ValidHandle()
 
     [System.Runtime.InteropServices.ComVisible(true)]
 #if FEATURE_REMOTING
-    public sealed class RegisteredWaitHandle : MarshalByRefObject {
+    public sealed class RegisteredWaitHandle : MarshalByRefObject
```

Comment:
Formating damage

Reply:
The braces don't match up in editor unless it understands if.

Reply:
Coreclr fixed this over a year ago, and much more. Perhaps we are overdue for an update, or perhaps coreclr isn't the reference and I should learn what is..

Reply:
eh, it is still in the referencesource actually.

Reply:
The reference source is more managed as well.